cdc/processor/processor.go

```diff
@@ -964,17 +965,6 @@ func (p *processor) createTablePipelineImpl(ctx cdcContext.Context, tableID mode
 		sink,
 		p.changefeed.Info.GetTargetTs(),
 	)
-	p.wg.Add(1)
-	p.metricSyncTableNumGauge.Inc()
-	go func() {
-		table.Wait()
-		p.wg.Done()
```

Comment:
Can we ensure table pipeline has exited after removing these lines

Reply:
yes, we call `table.Wait()` after we remove a table